airflow/models/dag.py

```diff
@@ -570,7 +570,7 @@ def date_range(
         message = "`DAG.date_range()` is deprecated."
         if num is not None:
             warnings.warn(message, category=DeprecationWarning, stacklevel=2)
-            return utils_date_range(start_date=start_date, num=num)
+            return utils_date_range(start_date=start_date, num=num, delta=self.normalized_schedule_interval)
```

Comment:
Since `normalized_schedule_interval` is also deprecated, this emits two successive warnings in one call. The `normalized_schedule_interval` warning should be suppressed here.

Reply:
Thank you for the review. I added the `catch_warnings` clause to the whole `return` line, because I also noticed that `utils_date_range` was producing yet another deprecation warning. Now it works correctly and using this method emits only one warning. Let me know if that is good now, thanks!